src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/IndexConfiguration/AbstractPimCatalogIntegration.php

```diff
@@ -17,20 +18,25 @@
  */
 abstract class AbstractPimCatalogIntegration extends TestCase
 {
-    /** TODO: Also could be generated from configuration */
-    const INDEX_NAME = 'product_index_test';
-
-    /** TODO: Maybe get this from configuration ? */
-    const PRODUCT_TYPE = 'pim_catalog_product';
+    const INDEX_TYPE = 'pim_catalog_product';
```

Comment:
It's not an index type, but a document type. You can have several types in one index.